### PR TITLE
UPS-3726 date format example

### PR DESCRIPTION
**Pull request by erwin1**
### Changed

- dateOfBirthFrom/To description with explicit date format

### CODE REVIEW

**Review by JurgenG — COMMENTED** [review]
Looks fine.

File: reference/UiTPAS.v2.json [hunk 1]
```diff
@@ -1148,7 +1148,7 @@
             },
             "in": "query",
             "name": "dateOfBirthTo",
-            "description": "Include only passholders with a dateOfBirth of this value (including) or older"
+            "description": "Include only passholders with a dateOfBirth of this value (including) or older. Format yyyy-mm-dd e.g. 2003-01-01"
```
Comment (bertramakers) [inline reference/UiTPAS.v2.json:1151]: You can make this a regex and make it part of the schema to be 100% safe. (`"format": "date"` accepts any valid ISO-8601 date/time notation if I'm not mistaken)

Comment (bertramakers) [reply reference/UiTPAS.v2.json:1151]: See https://swagger.io/docs/specification/data-models/data-types/#pattern

Comment (bertramakers) [reply reference/UiTPAS.v2.json:1151]: An example of a regex could be `^\d{4}-\d{2}-\d{2}$`

Comment (bertramakers) [reply reference/UiTPAS.v2.json:1151]: Nevermind ignore my comments, `"format": "date"` only accepts dates in the format that you described in the description. 👍

**Pull request merged**